packages/now-cli/src/util/dev/server.ts

```diff
@@ -826,6 +832,13 @@ export default class DevServer {
     // Wait for "ready" event of the watcher
     await once(this.watcher, 'ready');
 
+    // Configure the server to forward WebSocket "upgrade" events to the proxy.
+    this.server.on('upgrade', (req, socket, head) => {
+      this.proxy.ws(req, socket, head, {
+        target: `http://localhost:${this.devProcessPort}`,
```

Comment:
I'm also wondering about setting `target` here like this… I copied it from a `proxyPass` call site.

I think it could also be interesting to configure this as the default `target`, and have other calls _override_ the `target` option when they want to? I didn't look too deeply into what the override cases are doing though, or when `devProcessPort` is initialized (so I'm not actually sure configuring this at construction time would be possible).